xdsl/dialects/x86/x86.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+
+from xdsl.ir import Dialect
+
+from .register import GeneralRegisterType, Registers
+
+# This is where all of the instructions will be placed.
+Registers = Registers  # here to allow the test file to import the Registers class
```

Comment:
no need to do this at all

Reply:
without it the test complains because "module 'xdsl.dialects.x86' has no attribute 'Registers'". Probably an import issue

Reply:
that's because Registers needs to be imported in the `__init__.py`